src/main/java/com/ksm/wstbackoffice/exception/ControllerExceptionHandler.java

```diff
@@ -0,0 +1,23 @@
+package com.ksm.wstbackoffice.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+
+import java.util.Date;
+
+@ControllerAdvice
+public class ControllerExceptionHandler {
+    @ExceptionHandler(ResourceNotFoundException.class)
+    public ResponseEntity<ErrorResponseMessage> resourceNotFoundException(ResourceNotFoundException ex, WebRequest request) {
+        ErrorResponseMessage message = new ErrorResponseMessage(
+                HttpStatus.NOT_FOUND.value(),
```

Comment:
Do you think it's a good idea to put this value in the message if you have it in the response as an HTTP response code?

Reply:
removed HttpStatus.NOT_FOUND.value()